Abot2/Poco/CrawlConfiguration.cs

```diff
@@ -223,6 +224,11 @@ public CrawlConfiguration()
         /// </summary>
         public int MaxRobotsDotTextCrawlDelayInSeconds { get; set; }
 
+        /// <summary>
+        /// If true, will follow through on URLs found in meta tags.
+        /// </summary>
+        public bool FollowMetaRedirects { get; set; }
```

Comment:
Can we rename this to follow the common pattern of boolean settings? IsFollowMetaRedirectsEnabled